src/App.js

```diff
@@ -13,15 +15,15 @@ export default function App() {
                 <Navigation></Navigation>
 
                 <Switch>
-                    <Route path="/contact">Contact Page</Route>
+                    <Route path="/contact" component={ContactUs} />
                     <Route path="/meetdrbergin" component={Meetdrbergin} />
-
                     <Route path="/">
                         <Home></Home>
                     </Route>
                 </Switch>
 
-                <ContactInfo></ContactInfo>
+                <Route render={({ location }) => location.pathname !== "/contact" ? (<ContactInfo />) : null} />
+                <CopyrightInfo></CopyrightInfo>
```

Comment:
IMO the copy right info is too small to break apart into its own component, but this is totally fine!